app/views/SelectListView.tsx

```diff
@@ -179,7 +179,7 @@ class SelectListView extends React.Component<ISelectListViewProps, ISelectListVi
 			<>
 				<List.Separator />
 				<List.Item
-					title={item.name}
+					title={item.name ?? ''}
```

Comment:
Reviewed `something ?? ''` a lot this week :eyes:
Besides the wrong usage of `??`, I think it's a bad practice to enforce an empty string to make ts happy, if that's the reason.